mod_app/models/film_model.py

```diff
@@ -22,164 +34,129 @@ def __str__(self):
 
-    actors = models.ManyToManyField(
-        "Actor", related_name="films", blank=True, verbose_name="Cast"
+    cast = models.TextField(
+        blank=True,
+        null=True,
     )
-    crew = models.ManyToManyField("Crew", related_name="films", blank=True)
+    crew = models.TextField(blank=True, null=True, verbose_name="Credits")
 
-    intertitle_text = models.CharField(max_length=255, blank=True, null=True)
-    intertitle_photo = models.ImageField(
-        upload_to="images/intertitles/", blank=True, null=True
+    video = models.ForeignKey(
+        FileLink,
+        help_text="Link or upload the video file",
+        blank=True,
+        null=True,
+        on_delete=models.SET_NULL,
+        related_name="video_link",
     )
 
     # Technical section
 
     duration = models.IntegerField(
         blank=True,
         null=True,
-        help_text="Enter the run time in minutes.",
+        help_text="Enter the original run time in minutes.",
     )
     current_length = models.IntegerField(
         blank=True,
         null=True,
-        help_text="Enter the run time in minutes.",
+        help_text="Enter the run time of the BFI copy in minutes.",
     )
-
-    ELEMENT_CHOICES = [
-        ("pos", "Scene positive"),
-        ("ctn", "Negative coutertype"),
-        ("intn", "Internegative"),
-        ("lav", "Intermediate positive scene (lavender)"),
-        ("olay", "Titles"),
-    ]
-
-    element = models.CharField(
-        max_length=4,
-        choices=ELEMENT_CHOICES,
-        default="pos",
-    )
-
     support = models.CharField(
         max_length=1,
-        choices=[("S", "Safety"), ("N", "Nitrate")],
-        default="S",
-    )
-
+        choices=[("V", "Viewable"), ("M", "Master")],
+        default="V",
+    )
+    FORMAT_CHOICES = {
+        (9.5, "9.5 mm"),
+        (16, "16 mm"),
+        (35, "35 mm"),
+        (70, "70 mm"),
+    }
     format_type = models.CharField(
         max_length=255, blank=True, null=True, verbose_name="format"
-    )
-
-    rollers = models.IntegerField(
-        blank=True,
-        null=True,
-        verbose_name="Number of rollers",
-    )
-
+    )  # use choices + other
     is_in_colour = models.BooleanField(
         default=False,
         verbose_name="in colour?",
         help_text="Check box in colour and leave blank if black and white (default)",
     )
-    collection = models.CharField(
-        max_length=255,
-        blank=True,
-        null=True,
-    )  # could do this as choices potentially, maybe put on copy model
-
-    copies = models.ManyToManyField(
-        Copy,
-        help_text="Links to where the copies can be found",
-        related_name="copies",
+    print_comments = models.TextField(
         blank=True,
+        help_text="Optional notes about the print/s",
+        verbose_name="Notes on Prints",
     )
 
-    entry_date = models.DateField(blank=True, null=True)  # maybe put on copy model
-
     # Non filmic section / extras
```

Comment:
Nice that you've got these comments dividing it into sections. Why not go a step further an align it with the admin layout?

Reply:
I thought I had, unless I'm misunderstanding what you mean